lib/vm.py

```diff
@@ -361,6 +365,24 @@ def test_snapshot_on_running_vm(self):
         finally:
             snapshot.destroy(verify=True)
 
+    def vdi_resize(self, vdi_uuid, new_size=None):
+        # Reference
+        # MAX_VHD_SIZE = 2088960 * 1024 * 1024 # 2088960 MB
+        # MIN_VHD_SIZE = 1 * 1024 * 1024 # 1MB
+
+        # Following assert may be removed in future if we support online vdi resize
+        assert not self.is_running(), f"Expected halted, got running"
+
+        vdi_obj = VDI(vdi_uuid, sr=self.get_sr())
```

Comment:
just `vdi` would be suitable, no need for `_obj`

Reply:
done